lib/src/base-styles/_nx-card.scss

```diff
@@ -107,75 +78,64 @@
   }
 
   .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
+    align-self: center;
+    grid-area: card-content;
     margin: 0;
     padding: 0;
+    row-gap: 0;
   }
 
   .nx-card__call-out {
-    -ms-grid-row-align: center;
     align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 1;
     grid-area: card-callout;
-    -ms-grid-column-align: center;
     justify-self: center;
 
     align-items: center;
     border-right: $nx-border-default;
     display: flex;
     height: 100%;
-    margin: 0 $nx-spacing-xxl 0 0;
-    padding: 0 $nx-spacing-xxl 0 0;
+    margin: 0 $nx-spacing-l 0 0;
+    padding: 0 $nx-spacing-l 0 0;
   }
 
-  .nx-card__footer {
+  .nx-card__actions {
     box-sizing: content-box;
-    -ms-grid-row-align: center;
     align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 3;
-    grid-area: card-footer;
-    -ms-grid-column-align: start;
+    grid-area: card-actions;
     justify-self: start;
 
     @include container-horizontal;
     align-items: center;
-    border: none;
     border-left: $nx-border-default;
     display: flex;
     height: 100%;
-    margin: 0 0 0 $nx-spacing-xxl;
-    padding: 0 0 0 $nx-spacing-xxl;
+    margin: 0 0 0 $nx-spacing-l;
+    padding: 0 0 0 $nx-spacing-l;
```

Comment:
This isn't in sync with the mockups.  IIRC we agreed with @dturner32 to decrease these spacings?  He should update zeplin then.

Reply:
> This isn't in sync with the mockups. IIRC we agreed with @dturner32 to decrease these spacings? He should update zeplin then.

@dturner32 has adjusted the mockups